api/request_handler_test.go

```diff
@@ -0,0 +1,40 @@
+package api_test
+
+import (
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/gorilla/websocket"
+	"github.com/stretchr/testify/assert"
+	"github.com/topfreegames/pitaya-admin/api"
+	"github.com/topfreegames/pitaya-admin/request"
+	pt "github.com/topfreegames/pitaya-admin/testing"
+)
+
+func TestRequestHandler(t *testing.T) {
```

Comment:
It would be good to have a test which returns and output, maybe `connector.getsessiondata` after the `connector.setsessiondata`?